test/echo-tcp-client/main.cpp

```diff
@@ -41,6 +36,7 @@ void terminate(bool status, TCPEchoClient* client);
 
 char out_buffer[] = "Hello World\n";
 char buffer[256];
+char out_success[] = "{{success}}\n{{end}}\n";
```

Comment:
Moved it outside from onRx() in original code to sit with other strings. It indicates to the host test that target test has validated a round trip (echo) data and now terminating. It is good stop host test from echoing back as target client closes the connection right after sending this string. 
But it should not be confused with serial k,v command.